docs/CONTRIBUTORS

```diff
@@ -144,3 +144,4 @@ Yutetsu TAKATSUKASA
 Yu-Sheng Lin
 Yves Mathieu
 Zhanglei Wang
+ruki
```

Comment:
We require your legal name here, not a username, thanks.  Note this means you'll need to update it in github also.

If you have privacy concerns, as this is a trivial fix with no copyright worries, I can alternatively commit it myself.

Reply:
ok, you can help me to commit it.